### PR TITLE
DX12: using better approximation for constant color/alpha blending

### PR DESCRIPTION
**Pull request by raven02**

### CODE REVIEW

**Comment by raven02** [comment]
@kd-11 , does it look okay to you ?

File: rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp [hunk 1]
```diff
@@ -38,16 +38,21 @@ D3D12_BLEND get_blend_factor(u16 factor)
...
-	case CELL_GCM_CONSTANT_COLOR: return D3D12_BLEND_DEST_COLOR;
-	case CELL_GCM_ONE_MINUS_CONSTANT_COLOR: return D3D12_BLEND_INV_DEST_COLOR;
-	case CELL_GCM_CONSTANT_ALPHA: return D3D12_BLEND_DEST_ALPHA;
-	case CELL_GCM_ONE_MINUS_CONSTANT_ALPHA: return D3D12_BLEND_INV_DEST_ALPHA;
-		break;
+	case CELL_GCM_CONSTANT_COLOR:
+	case CELL_GCM_CONSTANT_ALPHA:
+	{
+		return D3D12_BLEND_ONE;	
+		LOG_ERROR(RSX,"Constant color/alpha blending detected and using BLEND_ONE approximation");
```
Comment (paulsapps) [inline rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp]: ??? all of these are unreachable, should be before the return statement

Comment (raven02) [inline rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp]: Yes , right .

Comment (kd-11) [inline rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp]: Nice catch. Didn't think to look too closely

**Comment by raven02** [comment]
Humm "Cannot add PPA: 'ppa:ubuntu-toolchain-r/test' , any insight?

**Pull request merged**